examples/demo/gloo/imshow_cuts.py

```diff
@@ -170,11 +174,11 @@ def on_mouse_move(self, event):
         y_baseline[...] = (xf, -1), (xf, -1), (xf, 1), (xf, 1)
 
         x_profile[1:-1, 0] = np.linspace(-1, 1, 512)
-        x_profile[1:-1, 1] = yf+0.15*I[y, :]
+        x_profile[1:-1, 1] = yf+0.15*I[y_norm, :]
```

Comment:
...otherwise later you get warnings here about non-integer indices that they say will become errors eventually